run-fe-ut.sh

```diff
@@ -110,24 +116,25 @@ fi
 
 mkdir ut_ports
 
+if [[ $FILTER_TEST != "" ]];then
+    FILTER_TEST=`echo $FILTER_TEST | sed -E 's/([^,]+)/!\1/g'`
+    TEST_NAME="$TEST_NAME, $FILTER_TEST"
```

Comment:
is this space intended?

Reply:
Yes, run like this:
`sh run-fe-ut.sh --run --filter com.starrocks.sql.plan.HiveTPCHPlanTest#testTPCH10, com.starrocks.sql.plan.HiveTPCHPlanTest#testTPCH18`

Reply:
@wanpengfei-git

Reply:
> Yes, run like this: `sh run-fe-ut.sh --run --filter com.starrocks.sql.plan.HiveTPCHPlanTest#testTPCH10, com.starrocks.sql.plan.HiveTPCHPlanTest#testTPCH18`

with multi filter cases, the tests should be enclosed in quotation marks, `sh run-fe-ut.sh --run --filter "com.starrocks.sql.plan.HiveTPCHPlanTest#testTPCH10, com.starrocks.sql.plan.HiveTPCHPlanTest#testTPCH18"`